.github/workflows/pr-auto.yml

```
@@ -17,16 +17,25 @@ jobs:
+      - name: 'bug' 라벨 추가
+        if: startsWith(github.event.pull_request.title, 'Fix') || startsWith(github.event.pull_request.title, 'fix')
         uses: actions-ecosystem/action-add-labels@v1
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          # PR 제목에 따라 라벨 자동 지정
-          labels: |
-            ${{ github.event.pull_request.title.startsWith('[fix]') && 'bug' || 
-               github.event.pull_request.title.startsWith('[chore]') && 'chore' || 
-               github.event.pull_request.title.startsWith('[feat]') && 'feature' || 
-               github.event.pull_request.title.startsWith('[refactor]') && 'refactor' }}
+        with: { labels: bug }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)